src/components/Pagination/index.jsx

```diff
@@ -60,80 +61,82 @@ const Pagination = ({
   const shouldDisplayPrev = currentPageIndex === 1;
   const shouldDisplayNext = currentPageIndex === totalPages;
   return (
-    <div className="pagination pagination-container">
-      <a
-        className={`btn btn-secondary btn-small pagination-prev ${
-          shouldDisplayPrev ? "is-disabled" : ""
-        }`}
-        onClick={handlePrevClicked}
-        onKeyDown={handlePrevClicked}
-        role="button"
-        tabIndex="0"
-      >
-        Prev
-      </a>
-      {shouldDisplayFirstPage && (
-        <span>
-          <a
-            className="pagination-link"
-            onClick={updatePageIndex}
-            onKeyDown={updatePageIndex}
-            value={1}
-            role="button"
-            tabIndex="0"
-          >
-            {1}
-          </a>
-          <a className="pagination-ellipses">&hellip;</a>
-        </span>
-      )}
-      <div className="pagination-links">
-        {Array(endPage - startPage + 1)
-          .fill()
-          .map((_, i) => startPage + i)
-          .map(number => (
-            <a
-              key={number}
-              className={cn("pagination-link", {
-                "is-active": number === currentPageIndex,
-              })}
-              onClick={updatePageIndex}
-              onKeyDown={updatePageIndex}
-              value={number}
-              role="button"
-              tabIndex="0"
-            >
-              {number}
-            </a>
-          ))}
-        {shouldDisplayLastPage && (
+    <Column width={10} offset={2}>
+      <div className="pagination pagination-container">
+        <a
+          className={`btn btn-secondary btn-small pagination-prev ${
+            shouldDisplayPrev ? "is-invisible" : ""
```

Comment:
is-hidden works better as the naming and moreso aligns with display: none rather than visibility: hidden

Reply:
ah woops! i actually ended up changing it to visibility:hidden as having it as display:none shifted things around instead of hiding it :)